Gemfile

```diff
@@ -62,7 +62,7 @@ end
 group(:development, optional: true) do
   gem 'memory_profiler', require: false, platforms: [:mri]
   gem 'pry', require: false, platforms: [:ruby]
-  gem "racc", "1.5.2", require: false, platforms: [:ruby]
+  gem "racc", "1.5.2", require: false
```

Comment:
Can you try moving this dependency to the `test` group (and leave the `platforms: [:ruby]` omitted)?